src/analyz/mod.rs

```diff
@@ -36,10 +36,10 @@ use schema_ver::SCHEMA_VER;
 
 basic_json_enum_impl!(mir::BinOp);
 
-basic_json_enum_impl!(mir::NullOp);
+basic_json_enum_impl!(mir::NullOp<'tcx>, 'tcx);
 basic_json_enum_impl!(mir::UnOp);
```

Comment:
These will likely need to be updated after https://github.com/GaloisInc/mir-json/pull/97 lands. (I can do this part.)